src/plugins/discover/public/context_awareness/__mocks__/index.tsx

```diff
@@ -171,3 +171,9 @@ export const createContextAwarenessMocks = ({
     profileProviderServices,
   };
 };
+
+export const createProfileProviderServicesMock = () => {
```

Comment:
```suggestion
const createProfileProviderServicesMock = () => {
```
Nit: does this need to be exported?